proc-macros/src/render_client.rs

```diff
@@ -206,7 +206,10 @@ impl RpcDescription {
 				let param_names = extract_param_names(&signature.sig);
 				// Combine parameter names and values to pass them as parameters.
 				let params_insert = param_names.iter().zip(params).map(|pair| {
-					let name = pair.0;
+					let mut name = pair.0.to_owned();
+					if name.starts_with("r#") {
```

Comment:
you could use https://doc.rust-lang.org/std/string/struct.String.html#method.strip_prefix here instead